pymatgen/entries/compatibility.py

```diff
@@ -803,16 +803,14 @@ def __init__(
         )
 
 
-"""
-Note from Ryan Kingsbury (2022-10-14): MaterialsProject2020Compatibility inherits from Compatibility
-instead of CorrectionsList which came before it because CorrectionsList had technical limitations.
-When we did the new scheme (MP2020) we decided to refactor the base Compatibility class to not
-require CorrectionsList.
-
-This was particularly helpful for the AqueousCorrection class. The new system gives complete
-flexibility to process entries however needed inside the get_adjustments() method, rather than
-having to create a list of separate correction classes.
-"""
+# Note from Ryan Kingsbury (2022-10-14): MaterialsProject2020Compatibility inherits from Compatibility
+# instead of CorrectionsList which came before it because CorrectionsList had technical limitations.
+# When we did the new scheme (MP2020) we decided to refactor the base Compatibility class to not
+# require CorrectionsList.
+
+# This was particularly helpful for the AqueousCorrection class. The new system gives complete
+# flexibility to process entries however needed inside the get_adjustments() method, rather than
+# having to create a list of separate correction classes.
```

Comment:
converted string to real comment

Reply:
I'm not sure what the 'official' best practice on multi-line comments is. This seems fine to me though.

Reply:
I'm not 100% confident about the "official" multi-line comments either, and I would like to hear more comments. But here is some info I have so far:
From the [PEP-8 doc](https://peps.python.org/pep-0008/#comments) under the `comments` section, there is:

> [Block Comments](https://peps.python.org/pep-0008/#block-comments)
Block comments generally apply to some (or all) code that follows them, and are indented to the same level as that code. **Each line of a block comment starts with a # and a single space** (unless it is indented text inside the comment).
    Paragraphs inside a block comment are separated by a line containing a single #.

So I assume it might be slightly better (though it should work almost the same) to use a stack of "#" signs, instead of a "floating" string (which should be specific to docstrings from my understanding), meanwhile `pylint` would give me `String statement has no effectPylint [W0105:pointless-string-statement]` warning.

Reply:
i think both versions are fine here